jmxtrans-core/src/main/java/com/googlecode/jmxtrans/model/Server.java

```diff
@@ -221,11 +239,6 @@ public String getUrl() {
 	}
 
 	@JsonIgnore
-	public JMXServiceURL getJmxServiceURL() throws MalformedURLException {
```

Comment:
Instead of removing this method, we should actually start using it. If we encapsulate the JMX URL here, we can have a `ServerJob` class that is completely agnostic of where the JMX values are coming from. Much much cleaner ...